test/VaultSanityCheck.ts

```diff
@@ -0,0 +1,107 @@
+import { ethers, network, getNamedAccounts } from "hardhat";
+import { assert } from "chai";
+import { describe, it } from "mocha";
+import { CHAINID, NULL_ADDR, AURORA_USDC_POOL } from "../constants/constants";
+import RibbonThetaVaultLogic from "../deployments/aurora/RibbonThetaVaultLogic.json";
+import RibbonThetaVaultWNEARCall from "../deployments/aurora/RibbonThetaVaultWNEARCall.json";
+import RibbonThetaVaultAURORACall from "../deployments/aurora/RibbonThetaVaultAURORACall.json";
+import ManualVolOracle from "../deployments/aurora/ManualVolOracle.json";
+
+beforeEach(async () => {
+  if (network.config.chainId !== CHAINID.AURORA_MAINNET) {
+    return;
+  }
+});
+
+describe("Aurora - VaultSanityCheck", () => {
+  const vaultParams = {
+    near: {
+      nextOption: '0x5366316bB6cC27F33C88c27e03bD54757FC91E28',
+      annualizedVol: 106480000
+    },
+    aurora: {
+      nextOption: '0xcCDe3B57996C7771529feC52562e335C85366381',
+      annualizedVol: 106480000
+    },
+  };
+
+  let keeperSigner;
+
+  beforeEach(async () => {
+    await network.provider.request({
+      method: "hardhat_reset",
+      params: [
+        {
+          forking: {
+            chainId: CHAINID.AURORA_MAINNET,
+            blockNumber: 58901375,
+            jsonRpcUrl: process.env.AURORA_URI,
+          },
+        },
+      ],
+    });
+
+    const { keeper } = await getNamedAccounts();
+
+    await network.provider.request({
+      method: "hardhat_impersonateAccount",
+      params: [keeper],
+    });
+
+    keeperSigner = await ethers.provider.getSigner(keeper);
+
+  });
+
+  it("NEAR #rollToNextOption changes current option", async () => {
```

Comment:
```if (network.config.chainId !== CHAINID.AURORA_MAINNET) {
  it()
}```

Reply:
It made more sense to put this in the `beforeEach` block (inside `describe`).